README.md

```diff
@@ -144,24 +150,29 @@ solo init -t v0.54.0-alpha.4 -i node1,node2,node3 -n "${SOLO_NAMESPACE}" -s "${S
 Version			: 0.31.1
 Kubernetes Context	: kind-solo
 Kubernetes Cluster	: kind-solo
-Kubernetes Namespace	: solo
+Kubernetes Namespace	: undefined
```

Comment:
somehow the namespace is not output correctly here

Reply:
solo init no longer has a namespace option since there is no caching, that is why it is undefined.

Reply:
I've opened a GH Issue to collapse that when it is undefined.: https://github.com/hashgraph/solo/issues/721